### ISSUE TITLE
Preview grid cell price label accessibility

### ISSUE THREAD

**Issue by ulrikus** [finn-no/FinniversKit#90]
The price label accessibility of the preview grid cell (still) doesn't read the price correctly. I thought I fixed it i while ago, but apparently not. Maybe I fixed it in the FINN app and not here.

I think it might have something to do with the fact that it is supposed to be as generic as possible.
